setup

```diff
@@ -10,9 +10,9 @@ set -a
 #-------------------------------------------------------------------------------
 
 # This file should be sourced, not executed!
-if [ "$0" == "${BASH_SOURCE[0]}" ]; then
-  echo "WARNING: $0 should be sourced, not executed. Try: source $0"
-fi
+#if [ "$0" == "${BASH_SOURCE[0]}" ]; then
+#  echo "WARNING: $0 should be sourced, not executed. Try: source $0"
+#fi
```

Comment:
Why are these three lines commented out?

Reply:
The protection against being run as a stand-alone script (which won't work; it's a script whose purpose is to set environment variables for other scripts) is highly valuable. I see no good reason to remove it.